src/store/actions.js

```diff
@@ -155,7 +155,7 @@ export default {
 			// TODO: handle short/ending streams and show their last element as well
 			return Promise.all(
 				getters
```

Comment:
What's going on with this eslint comment? :p

Reply:
Ooh, `⏎↹↹↹↹↹` means new line and 4 tabs! Not very straightforward :p

Reply:
fixed